src/main/java/com/example/teampandanback/domain/user/Role.java

```diff
@@ -0,0 +1,15 @@
+package com.example.teampandanback.domain.user;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum Role {
+
+    GUEST("ROLE_GUEST", "손님"),
```

Comment:
```suggestion
    CREW("ROLE_CREW", "크루원"),
```
CREW 로 변경하는 것이 적합하다고 생각합니다 ( 한글식 표현은 조금 더 논의 해봐도 좋을 듯 합니다)

Reply:
이해하였습니다! 논의하겠습니다!